.travis.yml

```diff
@@ -0,0 +1,2 @@
+language: rust
+rust: stable
```

Comment:
So presumably we don't need the threading hack here any more since there is no global state, correct? If so, I like it :)